ql/src/test/results/clientpositive/llap/check_constraint.q.out

```diff
@@ -2519,23 +2519,23 @@ STAGE PLANS:
         Map 7 
             Map Operator Tree:
                 TableScan
-                  alias: t
+                  alias: tmerge
```

Comment:
How sure we are of these changes?
My first guess that this is an independent change/fix where we fix using the alias name, but I might be wrong.